instrumentation/grpc/pom.xml

```diff
@@ -114,7 +111,42 @@
       </plugin>
     </plugins>
   </build>
+
   <profiles>
+    <profile>
+      <id>release</id>
+      <properties>
+        <!-- grpc-java < 1.15 supports Java 6 https://github.com/grpc/grpc-java/issues/3961 Current versions support Java 7 even though TLS typically implies Java 8+ -->
```

Comment:
example of building java 1.6 only in a profile, same as zipkin and zipkin-reporter